samples/UpgradeStepSample/UpgradeStepSample/UpgradeStepSample.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
These should be netstandard2.0 in case we decide to host in vs

Reply:
That's a good point, but it will be a somewhat larger change to make this work on NetStandard because Microsoft.DotNet.UpgradeAssistant.Extensions loads extensions with AssemblyLoadContext. I'll merge this as-is for now and assign myself a follow-up issue to make all the upgrade steps netstandard.